### DISCUSSION TITLE
Decide on name of potentially new app format

### DISCUSSION THREAD

**Discussion by LinusCDE** [toltec-dev/organization#11] [General]
**Discussion** for names for a potentially new app format takes place **here**.

Please read issue #10 for all information regarding this.

**Comment by LinusCDE** [comment]
Added the two names we did in that [first meeting](https://github.com/toltec-dev/organization/blob/main/meetings/discussion-2020-01-11.md#name) on monday.

**Comment by Eeems** [comment]
@alistair23 please add discussion here. Text from your comment:
> +1 on following the https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html
> 
> It matches the Gnome format as well: https://developer.gnome.org/desktop-entry-spec/
> 
> Which means we can standardise on a common format (instead of inventing something new)

**Reply by alistair23** [reply]
Sorry, I typed it in the wrong window.
